frontend/src/utils/date-utils.test.ts

```diff
@@ -0,0 +1,49 @@
+import { generateDateItemsBetweenForRanges } from './date-utils';
+
+describe('Test buildDateItemsFromStartEndDates', () => {
+  test('should return empty', () => {
+    // Act
```

Comment:
I'm not sure these Act / Arrange comments are very telling. Let's find more descriptive comments or get rid of them.

Reply:
It's just a habit of separating unit test in 3 steps.
// Arrange // Act // Assert.
I always found it usefull to know each part at a glance.
But as you wish